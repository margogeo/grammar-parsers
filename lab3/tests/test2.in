a = 3
b = 3
print(f(a))
print(b)