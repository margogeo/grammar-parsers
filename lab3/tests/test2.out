O0OI0 = 3
IOOI00OO0OII0=OOOO0OIO0*I11I010O0
OIOI0 = 3
print(f(O0OI0))
II0I0OO1=OO1I01I00-5722
print(OIOI0)
